src/OpenTK/Math/Vector2d.cs

```diff
@@ -627,6 +631,118 @@ public static void Max(ref Vector2d a, ref Vector2d b, out Vector2d result)
 
         #endregion
 
+        #region ComponentMin
+
+        /// <summary>
+        /// Calculate the component-wise minimum of two vectors
+        /// </summary>
+        /// <param name="a">First operand</param>
+        /// <param name="b">Second operand</param>
+        /// <returns>The component-wise minimum</returns>
+        public static Vector2d ComponentMin(Vector2d a, Vector2d b)
```

Comment:
That's a good point. "Creates a new vector from the smallest of the corresponding components of the given vectors."? I'm hesitant to use "returns" - it's evident that the function returns something when it has a return value.